<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ecflow_ui</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>640</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>24</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewTab"/>
    <addaction name="actionNewWindow"/>
    <addaction name="separator"/>
    <addaction name="actionSaveSessionAs"/>
    <addaction name="actionManageSessions"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Ser&amp;vers</string>
    </property>
    <addaction name="actionManageServers"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Pa&amp;nels</string>
    </property>
    <addaction name="actionAddTreeWidget"/>
    <addaction name="actionAddTableWidget"/>
    <addaction name="actionAddInfoPanel"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionNotification"/>
    <addaction name="actionCommandOutput"/>
    <addaction name="actionServerLoadViewer"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuRefresh">
    <property name="title">
     <string>&amp;Refresh</string>
    </property>
    <addaction name="actionRefreshSelected"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="actionResetSelected"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuRefresh"/>
   <addaction name="menuServer"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="viewToolBar">
   <property name="windowTitle">
    <string>View toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSearch"/>
   <addaction name="actionManageServers"/>
  </widget>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/close.svg</normaloff>:/viewer/close.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/images/exit.svg</normaloff>:/viewer/images/exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/images/configure.svg</normaloff>:/viewer/images/configure.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences ...</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="text">
    <string>New &amp;tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="text">
    <string>New &amp;window</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh all servers  &amp;in tab</string>
   </property>
   <property name="toolTip">
    <string>Refresh &lt;b&gt;all servers&lt;/b&gt; in current tab</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>&amp;Reset all servers in tab</string>
   </property>
   <property name="toolTip">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionConfigureNodeMenu">
   <property name="text">
    <string>Configure Node &amp;Menu</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
   <property name="toolTip">
    <string>View contents</string>
   </property>
  </action>
  <action name="actionManageServers">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/manage_server.svg</normaloff>:/viewer/manage_server.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Manage servers ...</string>
   </property>
   <property name="toolTip">
    <string>Manage servers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAddInfoPanel">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/images/add_info.svg</normaloff>:/viewer/images/add_info.svg</iconset>
   </property>
   <property name="text">
    <string>Add &amp;info panel</string>
   </property>
  </action>
  <action name="actionAddTreeWidget">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/images/add_tree.svg</normaloff>:/viewer/images/add_tree.svg</iconset>
   </property>
   <property name="text">
    <string>Add &amp;tree view panel</string>
   </property>
   <property name="toolTip">
    <string>Add tree widget</string>
   </property>
  </action>
  <action name="actionAddTableWidget">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/images/add_table.svg</normaloff>:/viewer/images/add_table.svg</iconset>
   </property>
   <property name="text">
    <string>Add ta&amp;ble view panel</string>
   </property>
   <property name="toolTip">
    <string>Add table widget</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About EcflowUI ...</string>
   </property>
  </action>
  <action name="actionManageSessions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Manage sessions ...</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/search.svg</normaloff>:/viewer/search.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Search ...</string>
   </property>
   <property name="toolTip">
    <string>Start search dialogue</string>
   </property>
  </action>
  <action name="actionRefreshSelected">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/reload_green.svg</normaloff>:/viewer/reload_green.svg</iconset>
   </property>
   <property name="text">
    <string>R&amp;efresh selected server</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionResetSelected">
   <property name="text">
    <string>Reset &amp;selected server</string>
   </property>
   <property name="toolTip">
    <string>Reset &lt;b&gt;selected&lt;/b&gt; server in current tab</string>
   </property>
  </action>
  <action name="actionSaveSessionAs">
   <property name="text">
    <string>S&amp;ave session as...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionNotification">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/notification.svg</normaloff>:/viewer/notification.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Notifications ...</string>
   </property>
   <property name="toolTip">
    <string>Start notifications dialogue</string>
   </property>
  </action>
  <action name="actionCommandOutput">
   <property name="text">
    <string>Shell command &amp;output ...</string>
   </property>
   <property name="toolTip">
    <string>Start shell command output dialogue</string>
   </property>
  </action>
  <action name="actionServerLoadViewer">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/monitor.svg</normaloff>:/viewer/monitor.svg</iconset>
   </property>
   <property name="text">
    <string>Server load viewer ...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
